docker/Dockerfile

```diff
@@ -78,13 +78,14 @@ RUN pip install --no-cache-dir -r doc/requirements.txt
 FROM base-build-python AS build-wheel-lightning-qubit
 WORKDIR /opt/pennylane-lightning
 RUN pip uninstall -y pennylane-lightning
-RUN python setup.py build_ext --define="ENABLE_KOKKOS=OFF"
+RUN python setup.py build_ext
```

Comment:
no need to enforce ENABLE_KOKKOS =OFF?

Reply:
`ENABLE_KOKKOS` is deprecated since Lightning-Qubit does not use Kokkos anymore (and its purpose was not to control the backend).